android/assets/jsons/translations/template.properties

```diff
@@ -33,6 +37,9 @@ Requires [buildingName] to be built in the city =
 Requires [buildingName] to be built in all cities = 
 Provides a free [buildingName] in the city = 
 Requires worked [resource] near city = 
+Requires worked [resource] = 
+or [resource] = 
+or [resource] near city = 
```

Comment:
When does this appear?

Reply:
Well, let's see - the one building tied to three resources... Monastery? No. Mint? No. Stable!
![image](https://user-images.githubusercontent.com/63000004/124837170-9f867b80-df84-11eb-9bd5-b972d294dd7a.png)
old: 'csv' style new: separate lines to allow individual links. That way to tie them together with words might not be the best idea, but - well, I _tried_ to focus more on 'works', maybe even 'reliably', less on perfect looks. Giving each line the same original template would have been easier, sure.

Reply:
This is EXTREMELY language-specific. I highly doubt that all languages can fit this in.
Instead, I suggest the following. A non-link line, followed by the resource names and links.

- Requires a worked resource near the city:
- Horses
- Sheep
- Cattle

Reply:
I agree (Had more or less the same idea over the long weekend). That it's an 'or' not an 'and' is clear enough in english from the "a" worked resource, but that may not be immediately clear to translators - comments in template.properties do propagate keeping their place?